<?xml version="1.0" ?>
<robot name="chessboard">
  <link name="chessboard__board">
    <inertial>
      <mass value="0"/>
      <origin rpy="0  0  0" xyz="0  0  0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
    <collision name="chessboard__collision">
      <origin rpy="0      0     -1.5708" xyz="0  0  0"/>
      <geometry>
        <mesh filename="package://baxter_sim_examples/models/chessboard/chessboard.dae" scale="0.0625 0.0625 0.0625"/>
      </geometry>
    </collision>
    <visual name="chessboard__visual">
      <origin rpy="0      0     -1.5708" xyz="0  0  0"/>
      <geometry>
        <mesh filename="package://baxter_sim_examples/models/chessboard/chessboard.dae" scale="0.0625 0.0625 0.0625"/>
      </geometry>
    </visual>
  </link>
</robot>
